binding/web/README.md

```diff
@@ -0,0 +1,181 @@
+# Koala Binding for Web
+
+## Koala Speech-to-Text Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Koala is an on-device noise suppression engine. Koala is:
+
+- Private; All voice processing runs locally.
+- Cross-Platform:
+    - Linux (x86_64), macOS (x86_64, arm64), and Windows (x86_64)
+    - Android and iOS
+    - Chrome, Safari, Firefox, and Edge
+    - Raspberry Pi (4, 3) and NVIDIA Jetson Nano
+
+## Compatibility
+
+- Chrome / Edge
+- Firefox
+- Safari
+
+### Restrictions
+
+IndexedDB is required to use `Koala` in a worker thread. Browsers without IndexedDB support
+(i.e. Firefox Incognito Mode) should use `Koala` in the main thread.
+
+## Installation
+
+### Package
+
+Using `Yarn`:
+
+```console
+yarn add @picovoice/koala-web
+```
+
+or using `npm`:
+
+```console
+npm install --save @picovoice/koala-web
+```
+
+### AccessKey
+
+Koala requires a valid Picovoice `AccessKey` at initialization. `AccessKey` acts as your credentials when using Koala SDKs.
+You can get your `AccessKey` for free. Make sure to keep your `AccessKey` secret.
+Signup or Login to [Picovoice Console](https://console.picovoice.ai/) to get your `AccessKey`.
+
+### Usage
+
+For the web packages, there are two methods to initialize Koala.
+
+#### Public Directory
+
+**NOTE**: Due to modern browser limitations of using a file URL, this method does __not__ work if used without hosting a server.
+
+This method fetches the model file from the public directory and feeds it to Koala. Copy the model file into the public directory:
+
+```console
+cp ${KOALA_MODEL_FILE} ${PATH_TO_PUBLIC_DIRECTORY}
+```
+
+#### Base64
+
+**NOTE**: This method works without hosting a server, but increases the size of the model file roughly by 33%.
+
+This method uses a base64 string of the model file and feeds it to Koala. Use the built-in script `pvbase64` to
+base64 your model file:
+
+```console
+npx pvbase64 -i ${KOALA_MODEL_FILE} -o ${OUTPUT_DIRECTORY}/${MODEL_NAME}.js
+```
+
+The output will be a js file which you can import into any file of your project. For detailed information about `pvbase64`,
+run:
+
+```console
+npx pvbase64 -h
+```
+
+#### Koala Model
+
+Koala saves and caches your model file in IndexedDB to be used by WebAssembly. Use a different `customWritePath` variable
+to hold multiple models and set the `forceWrite` value to true to force re-save a model file.
+
+Either `base64` or `publicPath` must be set to instantiate Koala. If both are set, Koala will use the `base64` model.
+
+```typescript
+const koalaModel = {
+  publicPath: ${MODEL_RELATIVE_PATH},
+  // or
+  base64: ${MODEL_BASE64_STRING},
+
+  // Optionals
+  customWritePath: "koala_model",
+  forceWrite: false,
+  version: 1,
+}
+```
+
+#### Init options
+
+Set `processErrorCallback` to handle errors if an error occurs while enhancing audio.
+
+```typescript
+// Optional
+const options = {
+  processErrorCallback: (error) => {}
+}
+```
```

Comment:
I think you should just merge the contents of this section into the one below. There are a bit too many sections in this README